### PR TITLE
minor fix

### PR DESCRIPTION
**Pull request by daichengxin**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker` --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/generate_diann_cfg/meta.yml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 name: generate_diann_cfg
-description: A module to generate DIA-NN configure file.
+description: A module to generate a DIA-NN configuration files, based on input files and params.
```
Comment (jpfeuffer) [inline modules/local/generate_diann_cfg/meta.yml:2]:
```suggestion
description: A module to generate DIA-NN configuration files, based on input files and params.
```

File: modules/local/generate_diann_cfg/meta.yml [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 keywords:
     - configure
     - DIA-NN
 tools:
     - custom:
         description: |
-            A custom module for DIA-NN configure file.
+            A custom module to generate a DIA-NN configuration files from input files and params.
```
Comment (jpfeuffer) [inline modules/local/generate_diann_cfg/meta.yml:9]:
```suggestion
            A custom module to generate DIA-NN configuration files from input files and params.
```

File: modules/local/librarygeneration/main.nf [hunk 3]
```diff
@@ -17,7 +17,8 @@ process LIBRARYGENERATION {
     output:
     path "*_lib.tsv", emit: lib_splib
     path "versions.yml", emit: version
-    file "*"
+    path "report.log.txt", emit: log
+    path "*.speclib"
```
Comment (jpfeuffer) [inline modules/local/librarygeneration/main.nf:21]: Cool looks better. How are the speclib used later, if they do not have an emit: ?

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
My fault, when I suggested changes I didnt see that multiple config files are generated.

**Review by ypriverol — APPROVED** [review]

**Pull request merged**